screens/PostScreen.tsx

```diff
@@ -62,6 +74,24 @@ const PostScreen: React.FC<PostScreenprops> = ({ navigation }) => {
     }
   };
 
+  const getLocation = async () => {
+    try {
+      let { status } = await Location.requestForegroundPermissionsAsync();
+
+      if (status !== "granted") {
+        //eventually error about not getting Permission
+        return;
+      }
+
+      let location = await Location.getCurrentPositionAsync({});
+      setLatitude(location.coords.latitude);
+      setLongitude(location.coords.longitude);
+      setAccuracy(location.coords.accuracy);
+    } catch (error) {
+      console.error("Error requesting location permission:", error);
+    }
+  };
+
```

Comment:
ab hier sind mir drei Ssachen aufgefallen:
1. Kommentar in Zeile 106 rausnehmen
2. Texte und Platzhalter stets auf Englsich (z.B. Z.107, 132)
3. Für die Fehlerbehandliung haben wir doch extra eine Komponente (ErrorComp) -> bitte diese benutzen (auch hier kannst du dich an UserProfile.tsx orientieren)